app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerViewModel.kt

```diff
@@ -42,11 +42,19 @@ class QuestionPlayerViewModel @Inject constructor() : ObservableViewModel() {
 
   fun getCanSubmitAnswer(): ObservableField<Boolean> = canSubmitAnswer
 
+  fun getAnswerItemList(): ObservableList<StateItemViewModel> {
+    return if (isSplitView.get() == true) {
+      rightItemList
+    } else {
+      itemList
+    }
+  }
+
   fun getPendingAnswerWithoutError(
-    answerHandler: InteractionAnswerHandler
+    answerHandler: InteractionAnswerHandler?
```

Comment:
Alternatively, you could pass in a lambda, e.g. ``getPendingAnswer(answerHandlerRetriver: (List<StateItemViewModel>) -> InteractionAnswerHandler?): UserAnswer``

I think that will result in mostly the same original function staying present. The caller just needs to pass in a reference to the assembler's method which keeps the API about as simple as before for callers (which in turn reduces the amount of duplication needed to be maintained between explorations & questions).

Reply:
Okay - i think I have done this. Please let me know if something doesn't look right. In terms of manually testing this implementation, I am not exactly sure what the 'QuestionPlayer' is? I have gone through a few activities and played through some things manually and everything seems fine - but I am not sure what to rigorously test. I know by reading the Wiki that since QuestionPlayer is inside /topic it has something to do with the Topic page (so I played around with the tabs and clicked through things there as much as I could).

Reply:
To test the state fragment:
- HomeActivity -> First Test Topic -> Lessons -> First Story -> Prototype Exploration
- Play through the whole thing
- Then, play the 'Image Region Selection Exploration' since this covers one interaction not currently in the prototype

To test questions:
- HomeActivity -> Fractions -> Practice -> Select all of the skills -> Click start
- Play through a practice session
- Repeat a few times to get different questions

When testing, make sure to consider:
- Submitting multiple wrong answers (to verify that expand/collapse is working)
- Test the audio voiceovers with different languages for chapters/explorations (these aren't supported yet for practice sessions)
- Rotating the device
- Waiting 60 seconds or submitting multiple wrong answers in a row to get a hint bulb in the lower left (only fractions _lessons_ will have these currently--neither questions nor the prototype will trigger these)
- Navigating back and forth to previous states
- Exiting the lesson or practice session midway and returning to start from the beginning

Reply:
> To test the state fragment:
> 
> * HomeActivity -> First Test Topic -> Lessons -> First Story -> Prototype Exploration
> * Play through the whole thing
> * Then, play the 'Image Region Selection Exploration' since this covers one interaction not currently in the prototype

I actually can't get to the end of the Prototype Exploration - for some reason emulators I have tried haven't supported any drag & drop (even on branches that drag & drop has been confirmed to work) and I don't have access to an android device. Is there any way you could test this portion? 

> To test questions:

> * HomeActivity -> Fractions -> Practice -> Select all of the skills -> Click start
> * Play through a practice session
> * Repeat a few times to get different questions

Going through training mode, I am noticing collapsing isn't working + pictures are not showing up for questions that require them. I am assuming this is a bug with QuestionPlayer. I will look into it. But please let me know if this sounds like it might also be an emulator issue! 

> * Test the audio voiceovers with different languages for chapters/explorations (these aren't supported yet for practice sessions) 

The voiceovers don't seem to be working - this could also really easily be an emulator issue. What do you think? I am nervous to deep dive into bugs and later find out it was an emulator issue all along.

Reply:
I can test the prototype--let me know via chat when you'd like me to do that.

Past answers not collapsing in the question player is #511. Images should be working--can you double check with both ``develop`` and the Gradle builds on this branch to see if something broke? If develop is also showing the issue, try a version of develop from last week.

I can test voiceovers & images locally when you need me to. You're right that it might be an emulator issue.

Reply:
Perfect - I will message you then. 

In terms of the images + voiceovers - I tested the current develop as well as a version of develop from ~10 days ago and neither of them are displaying images on any of the emulators I have. I feel this probably means it is an emulator issue.

Reply:
Makes sense, let's follow up on chat then, thanks!